tensorflow/python/kernel_tests/math_ops/bincount_op_test.py

```diff
@@ -178,6 +178,22 @@ def test_invalid_inputs(self):
               weights=weights,
               binary_output=binary_output))
 
+  def test_negative_value(self):
+    with self.assertRaises(errors.InvalidArgumentError):
+      value_tensor = random_ops.random_uniform(
+          [3, 1], minval=-256, maxval=257, dtype=dtypes.int32)
+      value = array_ops.identity(value_tensor)
+      weights = None
+      minlength = 2
+      maxlength = 0
+      dtype = "float32"
+      axis = -1
+      binary_output = True
+      out = bincount_ops.bincount(
```

Comment:
Can you put most of the values directly in the function call?

And for the dtype, use `dtypes.float32` instead of the string.

Thanks!